### PR TITLE
Performance optimization: use StartAfter flag in file list scans

### PR DESCRIPTION
**Pull request by korotkov-aerospike**
StartAfter flag was implemented in FMWK-559 in lib.

### CODE REVIEW

File: pkg/service/storage/azure.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func (a *AzureStorageAccessor) supports(storage model.Storage) bool {
 }
 
 func (a *AzureStorageAccessor) createReader(
-	ctx context.Context, storage model.Storage, path string, isFile bool, filter Validator,
+	ctx context.Context, storage model.Storage, path string, isFile bool, filter Validator, from string,
```
Comment (filkeith) [inline pkg/service/storage/azure.go]: may be name it `startFrom` ?  (and in all other places)

Comment (korotkov-aerospike) [reply pkg/service/storage/azure.go]: maybeEven startScanFrom ?

Comment (filkeith) [reply pkg/service/storage/azure.go]: yep it would be great, because `from` is to abstract

**Review by filkeith — APPROVED** [review]
lgtm

**Pull request merged**